star-rating/assets/README.md

```diff
@@ -0,0 +1,3 @@
+# Assets
+
+This folder contains assets for the apps in the App Directory.
```

Comment:
I don't think we need a separate README for the assets folder. If you think this description is still necessary, you can put it in the root README :)

Reply:
What do you mean by normalize?